module/cyphersystem.js

```diff
@@ -836,112 +836,112 @@ function allInOneRollDialog(actor, pool, skill, assets, effort1, effort2, additi
     if (pool == "Might" && cost > actor.data.data.pools.might.value || pool == "Speed" && cost > actor.data.data.pools.speed.value || pool == "Intellect" && cost > actor.data.data.pools.intellect.value) {
       additionalSteps = Math.abs(additionalSteps);
       allInOneRollDialog(actor, pool, skillRating, assets, effort1, effort2, additionalCost, additionalSteps, stepModifier);
-      return ui.notifications.notify(`You don’t have enough ${pool} points.`);
+      return ui.notifications.notify(`${game.i18n.localize("CYPHERSYSTEM.NotEnoughPoint")} ${pool}.`);
```

Comment:
Note to self: Investigate nested translation stings.

Reply:
Answered previously

Reply:
Refactored translation